test/Spec/MLabsPAB/PreBalance.hs

```diff
@@ -30,15 +31,15 @@ tests =
     ]
 
 privateKey1 :: PrivateKey
-privateKey1 = Wallet.privateKey (Wallet.knownWallet 1)
+privateKey1 = Address.unPaymentPrivateKey . Wallet.paymentPrivateKey $ Wallet.knownMockWallet 1
 
 pkh1, pkh2 :: PubKeyHash
-pkh1 = Wallet.pubKeyHash (Wallet.knownWallet 1)
-pkh2 = Wallet.pubKeyHash (Wallet.knownWallet 2)
+pkh1 = Address.unPaymentPubKeyHash . Wallet.paymentPubKeyHash $ Wallet.knownMockWallet 1
+pkh2 = Address.unPaymentPubKeyHash . Wallet.paymentPubKeyHash $ Wallet.knownMockWallet 2
 
 addr1, addr2 :: Address
-addr1 = Ledger.pubKeyHashAddress pkh1
-addr2 = Ledger.pubKeyHashAddress pkh2
+addr1 = Ledger.pubKeyHashAddress (PaymentPubKeyHash pkh1) Nothing
+addr2 = Ledger.pubKeyHashAddress (PaymentPubKeyHash pkh2) Nothing
```

Comment:
Will this break if addr1 or addr2 have staking addresses? I assume this would probably relate to my comments above

Reply:
prebalancing would have no problem with this actually, but the Contract is not yet designed to handle addresses with StakePubKeys correctly (as noted in a previous comment). I suggest we worry about this in a separate PR, if that's alright.